Automattic/wp-calypso
# Activate use of Babel Plugins for translatable string extraction on the CI

#### Changes proposed in this Pull Request

* This extracts the CI changes from #28583
* As per [the discussion in that PR](https://github.com/Automattic/wp-calypso/pull/28583#pullrequestreview-177597620) we'll likely move the extraction to a separate CI step or into one that already builds the client, such as `icfy-stats`.

#### Testing instructions

1. Open the details for the `ci/circleci: lint-and-translate` check at the bottom of this PR.
2. Navigate to the "Artifacts" tab and download the `/tmp/translate/calypso-strings.pot`
3. Verify that the contents is sane and contains both Gutenberg and Calypso strings. Filesize is currently around 1.3M.
